apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/ConsumerController.java

```diff
@@ -119,6 +120,16 @@ public List<ConsumerRole> assignNamespaceRoleToConsumer(@PathVariable String tok
     return consumerService.assignNamespaceRoleToConsumer(token, appId, namespaceName);
   }
 
+  @GetMapping("/consumers/list")
+  @PreAuthorize(value = "@permissionValidator.isSuperAdmin()")
+  public List<Consumer> getConsumerList(Pageable page){
+    return consumerService.findAllConsumer(page);
+  }
 
+  @DeleteMapping(value = "/consumers/delete/by-appId")
```

Comment:
iMaintain uniform URL specifications  will be more cleaner.

/consumers/delete/by-appId => /consumers/by-appId?

Reply:
Similar to above`, /consumers/by-appId` is defined to get `ConsumerToken` , so not sure if it is appropriate to use `/consumers/by-appId `+ `@DeleteMapping `definition to delete Consumer

Reply:
That's all right, e.g. the `deleteAppNamespace ` and `findAppNamespace` in NamespaceController have the same path but different methods.

https://github.com/apolloconfig/apollo/blob/b229e005c3d70f3c3d0ed7175f832c3d7eb9cd95/apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/NamespaceController.java#L183-L204

Reply:
> the deleteAppNamespace and findAppNamespace in NamespaceController have the same path but different methods

This is understandable. However, because Consumer and ConsumerToken are not the same object instance, the above question arises? If everyone thinks there is no problem, I will consider changing it

Reply:
You are right. I think the path definition of `getConsumerTokenByAppId` should be `@GetMapping(value = "/consumer-tokens/by-appId")` instead of `@GetMapping(value = "/consumers/by-appId")`. However, as it's already there for years, we couldn't change it now.
As for the path definition of `deleteConsumers`, I think `@DeleteMapping(value = "/consumers/by-appId")` is ok as there is no need to delete consumer tokens alone.

Reply:
I have modified the above method definition problem together, and now it seems more intuitive